### ISSUE TITLE
Otunnel UDP or tcp or HTTP

### ISSUE THREAD

**Issue by surereddy** [ooclab/otunnel#12]
Hello sir is this UDP /tcp or http tunnel please

**Comment by gwind** [comment]
No, it's TCP now. The UDP would be support in the future if we have more real requirements.

**Comment by surereddy** [comment]
Thank you for the software.I would like use for VOIP where UDP is more reliable for communication.

**Comment by gwind** [comment]
Sorry about that UDP is not supported now :-)
